Yoti.Auth.Sandbox/DocScan/Request/Resource/Liveness/ZoomLivenessResource.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Newtonsoft.Json;
+
+namespace Yoti.Auth.Sandbox.DocScan.Request.Resource.Liveness
+{
+    public class ZoomLivenessResource : SandboxPayload
```

Comment:
This should live with the integration tests (web client)

Reply:
Moved in ddbb848